src/mapper.cpp

```diff
@@ -996,15 +1003,20 @@ Mapper::mems_pos_clusters_to_alignments(const Alignment& aln, vector<MaximalExac
                     for (auto& ref : xindex->node_positions_in_paths(id, is_rev)) {
                         auto& name = ref.first;
                         for (auto pos : ref.second) {
-                            cerr << name << (is_rev?"-":"+") << pos + offset << ",";
-                            //by_start_pos[chrom][pos+offset].push_back(make_pair(&mem, node));
+                            cerr << name << (is_rev?"-":"+") << pos + offset;
+                            cerr << "|" << id << (is_rev ? "-" : "+") << ":" << offset << ",";
                         }
                     }
                 }
                 cerr << mem.sequence() << " ";
             }
             cerr << endl;
         }
+    };
+
+    if (debug) {
```

Comment:
There is some tension between enabling quickly dropping into debug mode and saving the useless branching by using the `#ifdef`. However, I agree. It's best to enable the debugging at compile time.

This PR isn't in the best state. I'll clean this up. There are also a lot of extraneous debugging lines in the current state of things.